Dockerfile

```diff
@@ -1,28 +1,38 @@
 FROM ubuntu:18.04
 FROM httpd:2.4
+FROM python:3
 
 RUN groupadd container_users && useradd -ms /bin/bash -g container_users belmont
+RUN echo "Acquire::Check-Valid-Until \"false\";\nAcquire::Check-Date \"false\";" | cat > /etc/apt/apt.conf.d/10no--check-valid-until
```

Comment:
@pmazumda What is this echo command doing ?

Reply:
Validation when installing software-properties-common package